providers/openshift/provider.go

```diff
@@ -140,33 +140,34 @@ func (p *OpenShiftProvider) setCA(paths []string) error {
 	return nil
 }
 
+// encodeSARWithScope adds a "scopes" array to the SAR if it does not have one already, and outputs
+// as pretty encoding.
+func encodeSARWithScope(json *simplejson.Json) ([]byte, error) {
+	if len(json.Get("scopes").MustArray()) == 0 {
+		json.Set("scopes", []interface{}{})
+	}
+	return json.EncodePretty()
```

Comment:
Shouldn't we use `Encode` to make the proxy have to send less data to the API server?